tools/cldr-apps/src/main/java/org/unicode/cldr/web/api/Auth.java

```diff
@@ -82,10 +83,17 @@ public Response login(
             String userIP = WebContext.userIP(hreq);
             CookieSession session = null;
             if (!request.isEmpty()) {
-                UserRegistry.User user =
-                        CookieSession.sm.reg.get(request.password, request.email, userIP);
+                UserRegistry.User user;
+                try {
+                    user = CookieSession.sm.reg.get(request.password, request.email, userIP);
+                } catch (LogoutException e) {
+                    user = null;
+                }
+                if (user == null) {
+                    user = AuthSurveyDriver.createTestUser(request.password, request.email);
+                }
                 if (user == null) {
-                    return Response.status(403, "Login failed").build();
+                    throw new LogoutException();
```

Comment:
there's already a catch(LogoutException) below which returns `Response.status(403, "Login failed").build()`

if we returned that response directly here, the catch below would trigger an "unused" warning, and removing it would result in a many lines of code changing their indentation, and this change would look much larger than it really is